src/mam4xx/gas_chem.hpp

```diff
@@ -24,6 +24,46 @@ const Real rel_err = 1.0e-3;
 const Real high_rel_err = 1.0e-4;
 const int max_time_steps = 1000;
 
+KOKKOS_INLINE_FUNCTION
+void usrrxt(Real rxt[rxntot], // inout
+            const Real temp, const Real invariants[nfs], const Real mtot,
```

Comment:
It seems a little odd to me to have an input variable be `temp`, but I think I might devote more mental space to this than the average 🙃

Reply:
Perhaps this is our old friend the temperature and not everyone's favorite variable name. :-)

Reply:
I renamed this variables.